pcsx2/Docs/PCSX2_FAQ.md

```diff
@@ -105,46 +105,46 @@ If you are using an ISO image:
 If you are using a disc:
 
 1) Click the CDVD menu
-2) Check that "Plugin" mode is selected.
-3) Click "Plugin Menu", then "Plugin Settings". Verify your disc drive shows up in the menu.
+2) Check that "Disc" mode is selected.
+3) Go to "Drive Selector", then select your drive.
 4) Click the System menu, then Boot CDVD.
 
-### Question 17: What is the difference between Full and Fast boot
-Full boot will launch the PS2 BIOS, which will then launch the game. Some games need this for multi-language settings or times to work.
+### Question 17: What is Fast boot?
+Normally, Pcsx2 will launch the PS2 BIOS, which will then launch the game. Some games need this for multi-language settings or times to work.
 
 Fast boot will directly mount and launch the game without first launching the PS2 BIOS. Though slightly faster, this may cause some multi-language games to boot into the wrong language, or games that use the clock to give inaccurate times. 
```

Comment:
Not really true anymore in 1.7 as refraction changed this behavior.

Maybe mention 1.6 and below could do this wrong behavior?

Reply:
Yeah, I'll go ahead and revise that.

And seriously, Codacy thinks ending questions with a question mark is an issue of medium severity? Shouldn't it be analyzing code, not grammar?

Reply:
Codacy doesn't have a set format for FAQ so yeah but code styles aren't rules, they are pretty arbitrary made by people.

Reply:
I went ahead and updated it to say 1.6 and below on the fast boot issues.

Reply:
Also adding to this, the comment about the clock is wrong. The clock is always sourced from the host system, so mention of it can be removed.

Reply:
The only thing that would be different is that you would use Framelimiter Off / Turbo it would become increasingly different to host system until you reboot which resyncs the clock. But that is useless to the FAQ and more a sidenote.

Reply:
Alright, I'll get rid of the mention of the clock, then.